recipes/python-seabreeze/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "python-seabreeze" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/ap--/python-seabreeze.git
+  git_rev: python-seabreeze-v{{ version }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - seabreeze
+    - seabreeze.cseabreeze
+
+about:
+  home: http://github.com/ap--/python-seabreeze
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.md
+  summary: 'Python module for Ocean Optics spectrometer'
+
+extra:
+  recipe-maintainers:
+    - ap--
```

Comment:
Might want to quote your username. Just a little concerned that this gets parsed erroneously.